src/theme/ContactForm/form-container.js

```diff
@@ -34,10 +34,15 @@ const getForm = (formId, fields) => {
 
 const FormContainer = ({ children, fields, formId, subtitle, title }) => {
   useEffect(() => {
-    const script = document.createElement('script');
-    script.async = true;
-    script.src = '/js/forms.js';
-    document.body.appendChild(script);
+    const segmentScript = document.createElement('script');
```

Comment:
Segment loaded here since it's only dealing with form data. If this is loaded in the `head` it is only loaded once since this is a SPA. If a user landed on a page w/o a form then none of the events were attached so going to a page with a form after wouldn't do anything. It would have to be a hard refresh. Same issue for the ad blocker modal.

Reply:
Thanks for catching this